examples/nodejs-nestjs/src/protected/protected.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { ProtectedController } from './protected.controller.js';
+import { fixedWindow } from '@arcjet/node';
+import { ArcjetGuard } from '../arcjet/arcjet.guard.js';
+import { ConfigModule } from '@nestjs/config';
+import { ProtectedService } from './protected.service.js';
+
+@Module({
+    imports: [ConfigModule.forRoot()],
+    controllers: [ProtectedController],
+    providers: [
+        ProtectedService,
```

Comment:
Add a comment here explaining that you can add different / chained rules and link to our docs?